docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -182,6 +182,10 @@ children:
       url: Decorators_repository.html
       output: 'web, pdf'
 
+  - title: 'Error handling'
+    url: Error-handling.html
```

Comment:
The `.md` file is called `Errors-handling` whereas here we have `Error-handling.html`. I think the better option is to rename the `.md` file to align with this and the title of the page.

Reply:
+1. Please rename `docs/site/Errors-handling.md` => `docs/site/Error-handling.md` for consistency even though this will render fine as-is (or should) since the permalink in the file is correct.

Reply:
Thank you for spotting this. I think I started with `Errors.md`, then decided to go with "Error Handling" and forgot to switch the first word back to singular.

I am going to apply your suggestion.